### PR TITLE
feat(OverlayMode): Rename only-qam to overlay-mode, qam to quick bar

### PR DESCRIPTION
**Pull request by pastaq**
This will require an update to any plugin that used add_to_qam as that will now be add_to_quick_bar. We can add a temporary transitional func that includes a deprecation warning to make the transition smoother.

### CODE REVIEW

File: assets/themes/card_ui-darksoul.tres [hunk 1]
```diff
@@ -8,7 +8,6 @@
 [ext_resource type="StyleBox" uid="uid://b3ta5yb32jlnb" path="res://assets/styles/darksoul/context_bar_menu_panel.tres" id="4_v80py"]
 [ext_resource type="StyleBox" uid="uid://bar3btp2yiodw" path="res://assets/styles/darksoul/inside_panel.tres" id="5_smg3e"]
 [ext_resource type="StyleBox" uid="uid://b8agb31kxbsap" path="res://assets/styles/darksoul/main_menu_panel.tres" id="6_nlift"]
-[ext_resource type="StyleBox" uid="uid://daavpt58e7jlj" path="res://assets/styles/darksoul/quick_access_menu_panel.tres" id="7_1vum8"]
```
Comment (pastaq) [inline assets/themes/card_ui-darksoul.tres:11]: Not sure why this happened, need feedback on how to restore this properly

File: assets/themes/card_ui-dracula.tres [hunk 2]
```diff
@@ -10,7 +10,6 @@
 [ext_resource type="StyleBox" uid="uid://x0dxx5c8u5rv" path="res://assets/styles/dracula/context_bar_menu_panel.tres" id="4_wqh56"]
 [ext_resource type="StyleBox" uid="uid://c2xl0k66pybg4" path="res://assets/styles/dracula/main_menu_panel.tres" id="7_n80rp"]
 [ext_resource type="StyleBox" uid="uid://c0pprevg50iuy" path="res://assets/styles/dracula/game_launch_buttons_panel.tres" id="8_iqinc"]
-[ext_resource type="StyleBox" uid="uid://br2w1fnanqkfa" path="res://assets/styles/dracula/quick_access_menu_panel.tres" id="9_5igh1"]
```
Comment (pastaq) [inline assets/themes/card_ui-dracula.tres:13]: Same as before

File: core/global/plugin_loader.gd [hunk 3]
```diff
@@ -39,12 +39,12 @@ var SettingsManager := load("res://core/global/settings_manager.tres") as Settin
 var parent: PluginManager
 var logger := Log.get_logger("PluginLoader", Log.LEVEL.INFO)
 ## Dictionary of installed plugins on the root file system.
-var plugins := {} # {plugin_id: {plugin.name: "name", plugin.id: "id", store.tags: ["qam", "steam"], ...}
+var plugins := {} # {plugin_id: {plugin.name: "name", plugin.id: "id", store.tags: ["quck_bar", "steam"], ...}
```
Comment (pastaq) [inline core/global/plugin_loader.gd:42]: Typo

File: core/main.gd [hunk 4]
```diff
@@ -16,16 +16,26 @@ func _ready() -> void:
...
-	# Launch CardUI in only-qam mode
+	# Launch CardUI in overlay mode
 	if "--qam-only" in args or "--only-qam" in args:
-		get_tree().change_scene_to_file("res://core/ui/card_ui_only_qam/card_ui_only_qam.tscn")
+		print("[WARN] Deprecation Warning: --only-qam and --qam-only launch arguments are deprecated\
+		and will be removed in a future update. Use --overlay-mode instead.")
+		get_tree().change_scene_to_file("res://core/ui/card_ui_overlay_mode/card_ui_overlay_mode.tscn")
+		return
+	elif "--overlay-mode" in args:
+		get_tree().change_scene_to_file("res://core/ui/vapor_ui_ooverlay_mode/overlay_mode_main.tscn")
```
Comment (ShadowApex) [inline core/main.gd:38]:
```suggestion
		get_tree().change_scene_to_file("res://core/ui/card_ui_overlay_mode/card_ui_overlay_mode.tscn")
```

File: core/main.gd [hunk 5]
```diff
@@ -16,16 +16,26 @@ func _ready() -> void:
 	
 	# Launch old ui
 	if "--vapor-ui" in args:
-		# in only-qam mode
+		# in overlay mode
 		if "--qam-only" in args or "--only-qam" in args:
-			get_tree().change_scene_to_file("res://core/ui/vapor_ui_only_qam/only_qam_main.tscn")
+			print("[WARN] Deprecation Warning: --only-qam and --qam-only launch arguments are\
+			deprecated and will be removed in a future update. Use --overlay-mode instead.")
+			get_tree().change_scene_to_file("res://core/ui/vapor_ui_ooverlay_mode/overlay_mode_main.tscn")
```
Comment (ShadowApex) [inline core/main.gd:23]:
```suggestion
			get_tree().change_scene_to_file("res://core/ui/vapor_ui_overlay_mode/overlay_mode_main.tscn")
```

File: core/main.gd [hunk 6]
```diff
@@ -16,16 +16,26 @@ func _ready() -> void:
...
+			return
+		elif "--overlay-mode" in args:
+			get_tree().change_scene_to_file("res://core/ui/vapor_ui_ooverlay_mode/overlay_mode_main.tscn")
```
Comment (ShadowApex) [inline core/main.gd:26]:
```suggestion
			get_tree().change_scene_to_file("res://core/ui/vapor_ui_overlay_mode/overlay_mode_main.tscn")
```

**Review by ShadowApex — CHANGES_REQUESTED** [review]

**Review by ShadowApex — APPROVED** [review]

**Pull request merged**